packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -135,17 +135,82 @@ handler.cleanUpPullRequest = async function cleanUpPullRequest(
   reactionId: number,
   github: Context['github']
 ) {
-  await github.issues
-    .removeLabel({owner, repo, issue_number: prNumber, name: label})
-    .catch(logger.error);
-  await github.reactions
-    .deleteForIssue({
+  await github.issues.removeLabel({
+    owner,
+    repo,
+    issue_number: prNumber,
+    name: label,
+  });
+  await github.reactions.deleteForIssue({
+    owner,
+    repo,
+    issue_number: prNumber,
+    reaction_id: reactionId,
+  });
+};
+
+/**
+ * Check if PR has been merged, closed, or unlabeled, then remove from Datastore table
+ * @param owner type string
+ * @param repo type string
+ * @param prNumber type number
+ * @param label type string
+ * @param reactionId type number or null
+ * @param github type githup API surface from payload
+ */
+handler.checkIfPRIsInvalid = async function checkIfPRIsInvalid(
+  owner: string,
+  repo: string,
+  prNumber: number,
+  label: string,
+  reactionId: number,
+  url: string,
+  github: Context['github']
+) {
+  let pr;
+  let labels;
+
+  try {
+    pr = (
+      await github.pulls.get({
+        owner,
+        repo,
+        pull_number: prNumber,
+      })
+    ).data;
+  } catch (err) {
+    pr = undefined;
```

Comment:
I might leave this just in case the github API is unavailable for some reason, but we can confirm by checking to see if a label is on the PR in the next function.